### PR TITLE
EDU-1554 Show existing document categories

### PR DESCRIPTION
**Pull request by ahelmberger**
Closes https://educandu.atlassian.net/browse/EDU-1554

### CODE REVIEW

**Comment by ahelmberger** [comment]
Will re-open after refining the UI.

File: src/utils/routes.js [hunk 1]
```diff
@@ -23,6 +23,7 @@ const userProfilePrefix = '/user-profile/';
 const documentInputPrefix = '/doc-inputs/';
 const mediaLibraryPrefix = '/media-library/';
 const samlAuthLoginPrefix = '/saml-auth/login/';
+const documentCategoriesPrefix = '/doc-categories/';
```
Comment (adrianaluchian) [inline src/utils/routes.js:26]: In the controller I went with `document-categories`. I honestly would move away from `doc` and `rev` with new logic and stick with full names. I think we should have done the same with `doc-inputs`.
What are your thoughts?

**Comment by adrianaluchian** [comment]
Looks good 👍 

Some small things from me:

- the collapsible arrow and title don't look vertically aligned
Screenshot 2024-04-23 at 12 31 11

- I find it a bit too cluttered to have a one-line footer per category (with 3 different links in there, a bit hard to follow). How about 3 lines? We can afford it, I think.

File: src/utils/routes.js (hunk 1 above)
Comment (ahelmberger) [reply src/utils/routes.js]: Changed it to `document-categories` as agreed.

**Comment by ahelmberger** [comment]
> Looks good 👍
> 
> Some small things from me:
> 
> * the collapsible arrow and title don't look vertically aligned
> 
> Screenshot 2024-04-23 at 12 31 11
> * I find it a bit too cluttered to have a one-line footer per category (with 3 different links in there, a bit hard to follow). How about 3 lines? We can afford it, I think.

Both issue are adressed (by hacking into antd and putting the footer onto separate lines)

**Review by adrianaluchian — APPROVED** [review]
